### ISSUE TITLE
Error running unless cl loaded

### ISSUE THREAD

**Issue by ambihelical** [pronobis/helm-flyspell#6]
Unless I explicitly load cl via (require 'cl), the following error occurs on running helm-flyspell-correct:

```
 In `Options' source: `(lambda nil (lexical-let ((tmp word)) (helm-flyspell--option-candidates tmp)))'   (void-function lexical-let)
```

**Comment by blaenk** [comment]
I'm also experiencing this. Perhaps #4 may help?

**Comment by pronobis** [comment]
Yes, that should do it. I merged #4 now.
